CHANGELOG.md

```diff
@@ -7,10 +7,123 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
--   removed jsonschema dependency
--   emailDelivery and smsDelivery Ingredients added
--   SMTP service added for emailDelivery
--   Twilio service added for smsDelivery
+### Removed
+
+-   Removed `jsonschema` library dependency. It was used to check user inputs against various JSON schema objects.
+
+### Added
+
+-   `emailDelivery` user config for Emailpassword, Thirdparty, ThirdpartyEmailpassword, Passwordless and ThirdpartyPasswordless recipes.
+-   `smsDelivery` user config for Passwordless and ThirdpartyPasswordless recipes.
+-   `Twilio` service integartion for smsDelivery ingredient.
+-   `SMTP` service integration for emailDelivery ingredient.
+
+### Deprecated
+
```

Comment:
There are also changes to the user type in the input to the email sending function right?